physx.js

```diff
@@ -704,6 +704,114 @@ const physxWorker = (() => {
       throw new Error('raycastPhysicsArray error');
     }
   };
+  /*
+  // Input
+  float *positions,
+  unsigned int numPositions,
+  unsigned int *faces,
+  unsigned int numFaces,
+
+  // Plane
+  float *position,
+  float *quaternion,
+  float *scale,
+
+  // Output
+  float *outPositions,
+  unsigned int *numOutPositions,
+  unsigned int *outFaces,
+  unsigned int *numOutFaces
+  */
+  w.doCut = (positions,numPositions,faces,numFaces,position,quaternion,scale) => {
+
+    let currOffset = 0;
+    const positionsArray = new Float32Array(scratchStack.f32.buffer, scratchStack.f32.byteOffset, numPositions);
+    currOffset += numPositions * 4;
+
+    const facesArray = new Uint32Array(scratchStack.f32.buffer, scratchStack.u32.byteOffset + currOffset, numFaces);
+    currOffset += numFaces * 4;
+
+    const positionParam = new Float32Array(scratchStack.f32.buffer, scratchStack.f32.byteOffset + currOffset, 3 * 4);

```

Comment:
I think the length should be `3`. That is not a byte length. You cannot have only part of a Float32.